src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,175 @@
+
+    /// @notice Mapping: recipient => token => total amount claimed
+    function cumulativeClaimed(address recipient, IERC20 token) external view returns (uint256);
+
+    /// @notice the commission for all operators across all avss
+    function globalOperatorCommissionBips() external view returns (uint16);
+
+    /// @notice returns the hash of the leaf
+    function calculateLeafHash(ClaimsTreeMerkleLeaf calldata leaf) external view returns (bytes32);
+
+    /// @notice returns 'true' if the claim would currently pass the check in `processClaims`
+    function checkClaim(PaymentMerkleClaim calldata claim) external view returns (bool);
+
+    /// EXTERNAL FUNCTIONS ///
+
+    /**
+     * @notice Creates a new range payment on behalf of an AVS, to be split amongst the
+     * set of stakers delegated to operators who are registered to the `avs`
+     * @param rangePayments The range payments being created
+     * @dev Expected to be called by the ServiceManager of the AVS on behalf of which the payment is being made
+     * @dev The duration of the `rangePayment` cannot exceed `MAX_PAYMENT_DURATION`
+     * @dev The tokens are sent to the `claimingManager` contract
+     * @dev This function will revert if the `rangePayment` is malformed,
+     * e.g. if the `strategies` and `weights` arrays are of non-equal lengths
+     */
+    function payForRange(RangePayment[] calldata rangePayments) external;
+
+    /**
+     * @notice similar to `payForRange` except the payment is split amongst *all* stakers
+     * rather than just those delegated to operators who are registered to a single avs
+     */
+    function payAllForRange(RangePayment[] calldata rangePayment) external;
+
+    /**
+     * @notice Sets the permissioned `paymentUpdater` address which can post new roots
+     * @dev Only callable by the contract owner
+     */
+    function setPaymentUpdater(address _paymentUpdater) external;
+
+    /**
+     * @notice Sets the delay in timestamp before a posted root can be claimed against
+     * @param _activationDelay Delay in timestamp before a posted root can be claimed against
+     * @dev Only callable by the contract owner
+     */
+    function setActivationDelay(uint64 _activationDelay) external;
+
+    /**
+     * @notice Sets the global commission for all operators across all avss
+     * @param _globalCommissionBips The commission for all operators across all avss
+     * @dev Only callable by the contract owner
+     */
+    function setGlobalOperatorCommission(uint16 _globalCommissionBips) external;
+
+    /**
+     * @notice Sets the address of the entity that can claim payments on behalf of the account
+     * @param account The account whose recipient is being set
+     * @param recipient The address of the entity that can claim payments on behalf of the account
+     * @dev Only callable by the `account`
+     */
+    function setRecipient(address account, address recipient) external;
```

Comment:
did we decide on recipient vs claimer?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/488/commits/2a9e9d5e0f3c92f77bb06cef994ab21210310cfe